src/components/VLanguage/VLanguage.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <VSelectField
+    v-bind="$attrs"
+    v-model="locale"
+    field-id="language"
+    :choices="choices"
+    :blank-text="$t('language.language')"
+    :label-text="$t('language.language')"
+  >
+    <template #start>
+      <VIcon :icon-path="globeIcon" />
+    </template>
+  </VSelectField>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from '@nuxtjs/composition-api'
+
+import { useI18n } from '~/composables/use-i18n'
+
+import VIcon from '~/components/VIcon/VIcon.vue'
+import VSelectField, {
+  type Choice,
+} from '~/components/VSelectField/VSelectField.vue'
+
+import type { LocaleObject } from '@nuxtjs/i18n'
+
+import globeIcon from '~/assets/icons/globe.svg'
+
+/**
+ * We put a little extra information in the Vue-i18n `locales` field such as the
+ * locale's name and native name, which comes in use here.
+ */
+interface OpenverseLocale extends LocaleObject {
+  name: string
+  nativeName?: string
+}
```

Comment:
This seems like it should go in `typings` so that it is automatically available to all usages of `i18n` library in Openverse without having to duplicate this or do casts. Did you look into whether it was possible to do that?

Reply:
Fixed in e8a180c.